source/exe/README.md

```diff
@@ -0,0 +1,10 @@
+State				Action		  		Next State				Side Effects
```

Comment:
i'll leave this pending for now.

Reply:
actually the mermaid diagrams look pretty cool and I will make a note to try them next time, but I propose that what I have is sufficient, and I think it might take a while to get the mermaid diagram to express that cyclic behavior and all the annotations I added to the graphics.